docs/relational-databases/system-dynamic-management-views/sys-dm-io-pending-io-requests-transact-sql.md

```diff
@@ -35,7 +35,7 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
 |**io_completion_request_address**|**varbinary(8)**|Memory address of the IO request. Is not nullable.|  
 |**io_type**|**nvarchar(60)**|Type of pending I/O request. Is not nullable.|  
 |**io_pending_ms_ticks**|**bigint**|Internal use only. Is not nullable.| 
-|**io_pending**|**int**|Indicates whether the I/O request is pending or has been completed by Windows. An I/O request can still be pending even when Windows has completed the request, but [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] has not yet performed a context switch in which it would process the I/O request and remove it from this list. Is not nullable.|  
+|**io_pending**|**int**|Indicates whether the I/O request is pending (1) or has been completed by Windows (0). An I/O request can still be pending even when Windows has completed the request, but [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] has not yet performed a context switch in which it would process the I/O request and remove it from this list. Is not nullable. <br /> **Value** <br /> 0 = Pending SQL Server <br /> 1 = Pending Windows <br />|  

```

Comment:
@SeanGallardy  it occurs that this is a good opportunity to update this language to use Operating System not Windows, since I assume this DMV is supported and behaves similarly via the PAL in SQL on Linux.  Can you confirm?

```suggestion
|**io_pending**|**int**|Indicates whether the I/O request is pending (1) or has been completed by the operating system (0). An I/O request can still be pending even when OS has completed the request, but [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] has not yet performed a context switch in which it would process the I/O request and remove it from this list. Is not nullable. <br /> **Value** <br /> 0 = Pending SQL Server <br /> 1 = Pending OS <br />|  

```

Reply:
Sounds good.